docs/Background-Jupyter.md

```diff
@@ -1,8 +1,9 @@
-# Jupyter
+# Background: Jupyter
 
 [Jupyter](https://jupyter.org) is a fantastic tool for writing code with 
-embedded visualizations. We provide one such notebook, `Basics.ipynb`, for 
-testing the Python API. 
+embedded visualizations. We provide one such notebook, `python/Basics.ipynb`, for 
+testing the Pyton control interface to a Unity build in the 
+[Getting Started with the 3D Balance Ball Environment](Getting-Started-with-Balance-Ball.md).
```

Comment:
This notebook is not meant to be only for this tutorial

Reply:
Ah, good catch - meant that this where we introduce it.